include/nbl/builtin/hlsl/tgmath.hlsl

```diff
@@ -27,6 +31,15 @@ bool isnan(uint64_t val)
 	return bool((ieee754::extractBiasedExponent<float64_t>(asFloat) == ieee754::traits<float64_t>::specialValueExp) && (val & ieee754::traits<float64_t>::mantissaMask));
 }
 
+// TODO: better implementation, also i'm not sure this is the right place for this function
+template<typename UINT>
+UINT lerp(UINT a, UINT b, bool c)
+{
+	return c ? b : a;
+}
+
+}
```

Comment:
maybe the glsl compat headers where we tried to add sqrt and friends? and call it `mix` ?

Reply:
I added a better implementation of mix/lerp on `master` recently